karate-junit5/src/main/java/com/intuit/karate/junit5/FeatureNode.java

```diff
@@ -94,4 +97,14 @@ public DynamicTest next() {
         return this;
     }
 
+    // fetch src uri to point to scenario in feature file.
+    public URI getFeatureSrcURI(ScenarioExecutionUnit sunit) {
```

Comment:
very minor nit-pick - but the naming convention I use is `getFeatureSrcUri`

Reply:
got it! Secondly I am thinking to push this method to Scenario class itself - as its path for that actual scenario - and possibly could be leveraged at other places. making the change and updating PR.

Reply:
good idea ! yes, add it to scenario